client/my-sites/domains/domain-management/edit/transfer.jsx

```diff
@@ -43,26 +44,76 @@ class Transfer extends React.PureComponent {
 		this.props.paymentSettingsClick( this.props.domain );
 	};
 
+	restartTransfer = () => {
+		const { domain, selectedSite, translate } = this.props;
+
+		restartInboundTransfer( selectedSite.ID, domain.name, ( error, result ) => {
+			if ( result ) {
+				this.props.successNotice( translate( 'The transfer has been successfully restarted.' ), {
+					duration: 5000,
+				} );
+				fetchDomains( selectedSite.ID );
+			} else {
+				this.props.errorNotice(
+					error.message || translate( 'We were unable to restart the transfer.' ),
+					{
+						duration: 5000,
+					}
+				);
+			}
+		} );
+	};
+
+	getCancelledContent() {
+		const { domain, translate } = this.props;
+
+		return (
+			<Card>
+				<div>
+					<p className="edit__transfer_text_fail">{ translate( 'Domain Transfer Failed' ) }</p>
```

Comment:
Hi! I've found a possible matching string that has already been translated **17** times:
`translate( 'Domain transfer failed' )` *ES Score: 17*
See 1 additional suggestion in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/update/failed-transfer-screen)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).